cli/test.js

```diff
@@ -23,49 +22,19 @@ function test(command, argv) {
   const specsPath = path.resolve(process.cwd(), 'src/app/**/*.spec.ts');
   const specsGlob = glob.sync(specsPath);
 
-  let lintResult;
-
-  const onRunStart = () => {
-    localeAssetsProcessor.prepareLocaleFiles();
-    lintResult = tsLinter.lintSync();
-  };
-
-  const onRunComplete = () => {
-    if (lintResult && lintResult.exitCode > 0) {
-      // Pull the logger out of the execution stream to let it print
-      // after karma's coverage reporter.
-      setTimeout(() => {
-        logger.error('Process failed due to linting errors:');
-        lintResult.errors.forEach(error => logger.error(error));
-      }, 10);
-    }
-  };
-
   const onExit = (exitCode) => {
-    if (exitCode === 0) {
-      if (lintResult) {
-        exitCode = lintResult.exitCode;
-      }
-    }
-
     logger.info(`Karma has exited with ${exitCode}.`);
     process.exit(exitCode);
   };
 
-  const onBrowserError = () => {
-    const stopper = require('karma').stopper;
-    stopper.stop({}, () => onExit(1));
-  };
-
   if (specsGlob.length === 0) {
     logger.info('No spec files located. Skipping test command.');
     return onExit(0);
   }
 
+  localeAssetsProcessor.prepareLocaleFiles();
```

Comment:
Curious what y'all think about running this method once, instead of before every test run.

Reply:
That would at least be consistent with how `skyux serve` works, although I think we have an issue to make that and other config json files handle live reloading.